Sources/Purchasing/StoreKitAbstractions/SK2StoreTransaction.swift

```diff
@@ -16,14 +16,17 @@ import StoreKit
 @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
 internal struct SK2StoreTransaction: StoreTransactionType {
 
-    init(sk2Transaction: SK2Transaction, jwsRepresentation: String) {
+    init(sk2Transaction: SK2Transaction, jwsRepresentation: String, environment: StoreEnvironment? = nil) {
```

Comment:
Needed a way to force the environment of a StoreTransaction since all transactions in tests have the `xcode` environment.